TypeCobol/Compiler/SqlScanner/SqlScanner.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using TypeCobol.Compiler.Scanner;
+
+namespace TypeCobol.Compiler.SqlScanner
+{
+    /// <summary>
+    /// A Sql Scanner related to a Scanner 
+    /// </summary>
+    public static class SqlScanner
+    {
+        /// <summary>
+        /// True if the given char can bepart of an SQL keyword.
```

Comment:
typo: "be part"

Reply:
done in [8e7eb2d](https://github.com/TypeCobolTeam/TypeCobol/pull/1928/commits/8e7eb2d5d8099a08412b896ce27756a1a43c4800)